packages/documentation/src/stories/components/dropdown/dropdown.docs.mdx

```diff
@@ -0,0 +1,60 @@
+import { Meta, Source } from '@storybook/blocks';
+import { PostTabs, PostTabHeader, PostTabPanel } from '@swisspost/design-system-components-react';
+import * as DropdownStories from './dropdown.stories';
+import StylesPackageImport from '../../../shared/styles-package-import.mdx';
+import NgbComponentDemoLink from '../../../shared/nb-bootstrap/ngb-component-demo-link.mdx';
+import NgbComponentAlert from '../../../shared/nb-bootstrap/ngb-component-alert.mdx';
+import NgbComponentImport from '../../../shared/nb-bootstrap/ngb-component-import.mdx';
+import SampleBasic from './dropdown-basic.sample.html?raw';
+import SampleDropup from './dropdown-dropup.sample.html?raw';
+import SampleDropupScroll from './dropdown-dropup-scroll.sample.html?raw';
+import SampleIcons from './dropdown-icons.sample.html?raw';
+
+<Meta of={DropdownStories} />
+
+<div className="d-flex align-items-center justify-content-between">
+  # Dropdown
+
+  <NgbComponentDemoLink component="dropdown" />
+</div>
+
+<p className="lead">The Dropdown component provides an interactive menu interface, allowing users to select a single option from a list of choices that expands when activated.</p>
```

Comment:
It would be nice if it was bit clearer. Especially the difference with the select. I think it's more about actionable item, but can you check with someone else?